### DISCUSSION TITLE
Get galaxy brain

### DISCUSSION THREAD

**Discussion by miladkhalighi** [Schweinepriester/github-profile-achievements#395] [Q&A] [answered]
Here we help each other to get it , lets go

**Comment by YSD98** [comment, marked as answer]
I want a galaxy brain Badge

**Comment by miladkhalighi** [comment]
@YSD98 done, mark me

**Reply by YSD98** [reply]
https://github.com/Schweinepriester/github-profile-achievements/discussions/396
I can mark u here on above link
